### PR TITLE
docs(README): add information about provenance file

### PR DESCRIPTION
**Pull request by developer-guy**
#### Summary

add information about provenance file

#### Ticket Link

Fixes

#### Release Note

```release-note
docs(README): add information about provenance file
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,8 @@ $ helm sigstore upload <path_to_packaged_chart>
 Created Helm entry at index 6821, available at: https://rekor.sigstore.dev/api/v1/log/entries/b30a142ef6c8b0480cd3e081fc99bc3d2a1a50ef60f68749c983a1479be6c4b9
 ```
 
-_NOTE_: The provenance file must be located in the same directory as the packaged chart
+_NOTE_: The provenance file must be located in the same directory as the packaged chart.
+> To generate a provenance file, please reach out the official documentation of [Helm Provenance and Integrity](https://helm.sh/docs/topics/provenance/).
```
Comment (sabre1041) [inline README.md:79]:
```suggestion
> To generate a provenance file, please consult the official documentation of [Helm Provenance and Integrity](https://helm.sh/docs/topics/provenance/).
```

**Review by sabre1041 — CHANGES_REQUESTED** [review]
Minor change requested

File: README.md (hunk 1 above)
Comment (developer-guy) [reply README.md]: sorry about my poor English 🤦🏻‍♂️ fixed.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**